plugins/workspace-minimap/src/minimap.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A minimap is a miniature version of your blocks that
+ * appears on top of your main workspace. This gives you an overview
+ * of what your code looks like, and how it is organized.
+ * @author cesarades@google.com (Cesar Ades)
+ */
+
+import * as Blockly from 'blockly/core';
+
+// Events that should be send over to the minimap from the primary workspace
+const BLOCK_EVENTS_ = [
+    Blockly.Events.BLOCK_CHANGE,
+    Blockly.Events.BLOCK_CREATE,
+    Blockly.Events.BLOCK_DELETE,
+    Blockly.Events.BLOCK_DRAG,
+    Blockly.Events.BLOCK_MOVE];
+
+/**
+ * A minimap is a miniature version of your blocks that appears on
+ * top of your main workspace. This gives you an overview of what
+ * your code looks like, and how it is organized.
+ * @param {!Blockly.WorkspaceSvg} workspace The workspace to sit in.
```

Comment:
Nit: Remove param comment from class. Params are only applied to methods!

Reply:
Ok, the backpack plugin also does this fyi

Reply:
Haha good to know, I'll file an issue to clean it up.